OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -349,4 +349,4 @@
   </Target>
   -->
   <ItemGroup />
-</Project>
+</Project>
```

Comment:
What needs to be reverted? Those changes in the project file were done by Visual Studio automatically when I renamed CreatesShroud.cs to CreatesConcealment.cs.